src/sentry/snuba/metrics/fields/snql.py

```diff
@@ -110,6 +157,15 @@ def sessions_errored_set(metric_ids, alias=None):
     )
 
 
+def all_transactions(org_id, metric_ids, alias=None):
+    return _dist_count_aggregation_on_tx_status_factory(
+        org_id,
+        exclude_tx_statuses=[],
```

Comment:
Python gotcha: never use `[]` or `{}` as default arguments: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

```suggestion
        exclude_tx_statuses=None,
```

Reply:
It's not a default argument, is it? It's not the definition of the function, just calling it.

Reply:
You're right, sorry. That's what I get for not taking a good look 😄 .

Reply:
I found that piece of code not straightforward to understand, but it was the best approach considering readability, generalization, and ease of use. However, if you have any suggestions, feel free to share them.